netlogo-gui/src/main/editor/EditorMenu.scala

```diff
@@ -0,0 +1,9 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.editor
+
+import javax.swing.Action
+
+trait EditorMenu {
+  def offerAction(action: Action): Unit
```

Comment:
It's definitely possible that it could be. At some point in the future I would like to enhance the system by making keyboard shortcuts configurable (and maybe making what operations appear configurable). For now, it lets us add actions to the menu bar and helps to reduce some of the dependency problems in `App.scala`.